src/test/java/com/puppycrawl/tools/checkstyle/gui/ParseTreeTablePModelTest.java

```diff
@@ -163,8 +163,109 @@ public void testJavadocChild() {
+
+        final ParseTreeTablePModel parseTree = new ParseTreeTablePModel(null);
+        final Object treeModel = parseTree.getValueAt(node, 0);
+        final String type = (String) parseTree.getValueAt(node, 1);
+        final int line = (int) parseTree.getValueAt(node, 2);
+        final int column = (int) parseTree.getValueAt(node, 3);
+        final String text = (String) parseTree.getValueAt(node, 4);
+
+        Assert.assertEquals("IDENT", type);
+        Assert.assertEquals(4, line);
+        Assert.assertEquals(6, column);
+        Assert.assertEquals("InputJavadocAttributesAndMethods", text);
+        Assert.assertNull(treeModel);
+
+        try {
+            parseTree.getValueAt(node, 231);
+            Assert.fail("IllegalStateException expected");
+        }
+        catch (IllegalStateException ex) {
+            Assert.assertEquals("Unknown column", ex.getMessage());
+        }
+
+    }
+
+    @Test
+    public void testGetValueAtDetailNode() {
+        final DetailAST commentContentNode = tree.getFirstChild().getNextSibling().getFirstChild();
+        final ParseTreeTablePModel parseTree = new ParseTreeTablePModel(null);
+        parseTree.setParseMode(ParseMode.JAVA_WITH_JAVADOC_AND_COMMENTS);
+        final Object child = parseTree.getChild(commentContentNode, 0);
+
+        Assert.assertFalse(parseTree.isLeaf(child));
+        Assert.assertTrue(parseTree.isLeaf(tree.getFirstChild()));
+
+        final Object treeModel = parseTree.getValueAt(child, 0);
+        final String type = (String) parseTree.getValueAt(child, 1);
+        final int line = (int) parseTree.getValueAt(child, 2);
+        final int column = (int) parseTree.getValueAt(child, 3);
+        final String text = (String) parseTree.getValueAt(child, 4);
+        final String expectedText = String.join("", System.lineSeparator(),
+            "* class javadoc", System.lineSeparator(), "<EOF>");
+
+        Assert.assertEquals(null, treeModel);
+        Assert.assertEquals("JAVADOC", type);
+        Assert.assertEquals(1, line);
+        Assert.assertEquals(0, column);
+        Assert.assertEquals(expectedText, text);
+
+        try {
+            parseTree.getValueAt(child, 6);
+            Assert.fail("IllegalStateException expected");
+        }
+        catch (IllegalStateException ex) {
+            Assert.assertEquals("Unknown column", ex.getMessage());
+        }
+
+    }
+
+    @Test
+    public void testColumnMethods() {
+        final ParseTreeTablePModel parseTree = new ParseTreeTablePModel(null);
+        Assert.assertEquals(ParseTreeTableModel.class, parseTree.getColumnClass(0));
+        Assert.assertEquals(String.class, parseTree.getColumnClass(1));
+        Assert.assertEquals(Integer.class, parseTree.getColumnClass(2));
+        Assert.assertEquals(Integer.class, parseTree.getColumnClass(3));
+        Assert.assertEquals(String.class, parseTree.getColumnClass(4));
+
+        try {
+            parseTree.getColumnClass(67);
```

Comment:
Would this work the same if we used `parseTree.getColumnCount()` instead of 67? That might make more sense then seeing some random number here.